test/resources/tektonconfigs.go

```diff
@@ -50,7 +51,9 @@ func EnsureTektonConfigExists(kubeClientSet *kubernetes.Clientset, clients confi
 	}
 
 	tcCR, err := clients.Get(context.TODO(), names.TektonConfig, metav1.GetOptions{})
-
+	// this timeout is needed to make sure that the e2e tests work
+	// TODO: https://github.com/tektoncd/operator/issues/401
+	time.Sleep(120 * time.Second)
```

Comment:
by any chance is this going to affect the overall time of our test?

Reply:
Oh, my bad. somehow I took it as 2 hrs.

Reply:
We should use the `wait` package (or function) from `tektoncd/pipeline` here. `time.Sleep` is very very flakey prone 😓

Reply:
practically no change was observed. 

This time out ensures that the test does not become flaky as sometimes, the tektonconfig instance will not be created by the time this line is executed. 

What we really need is a proper retry interval + timeout. (as described in the issue mentioned in the comment.)

Reply:
Yes, doing something similar to https://github.com/tektoncd/pipeline/blob/main/test/wait.go#L69 would help and make it even less flaky 😝

Reply:
shall we handle it after https://github.com/tektoncd/operator/pull/385 merges. This changes how we define `ready for testing` state.

Reply:
yes of course, thanks for linking the comment in the issue.